File: cs/libldt3/model/enums/GrenzwertindikatorErweitert.cs
using libldt3.attributes;
using libldt3.model.objekte;

namespace libldt3
{
    namespace model
    {
        namespace enums
        {
            [Objekt]
            public class GrenzwertindikatorErweitert
            {
                public Grenzwertindikator Value;
                [Feld(Value = "8126", Name = "Fehlermeldung_Aufmerksamkeit", Feldart = Feldart.bedingt_muss)]
                [Regelsatz(Laenge = 28)]
                public FehlermeldungAufmerksamkeit fehlermeldungAufmerksamkeit;
            }
        }
    }
}

File: java/src/main/java/libldt3/model/enums/GrenzwertindikatorErweitert.java
package libldt3.model.enums;

import libldt3.annotations.Feld;
import libldt3.annotations.Feldart;
import libldt3.annotations.Objekt;
import libldt3.annotations.Regelsatz;
import libldt3.model.Kontext;
import libldt3.model.objekte.FehlermeldungAufmerksamkeit;

@Objekt
public class GrenzwertindikatorErweitert implements Kontext {
    @SuppressWarnings("unused")
    public Grenzwertindikator value;
    @Feld(value = "8126", name = "Fehlermeldung_Aufmerksamkeit", feldart = Feldart.bedingt_muss)
    @Regelsatz(laenge = 28)
    public FehlermeldungAufmerksamkeit fehlermeldungAufmerksamkeit;
}